### DISCUSSION TITLE
(joke) bug report: my brain is cooked

### DISCUSSION THREAD

**Discussion by alicealysia** [YaLTeR/niri#752] [General]
I used to be satisfied with karousel and paperwm. Okay with just using a desktop environment with a scrolling plugin tacked on. Things were simpler then. I had all my settings, and apps all driven by a gui. Updating settings and config was a short and painless process, all my apps worked well, regardless of wayland or xorg.

But then I tried niri. And I learned what scrolling can look like. Now I can't go back. Despite all the software I'm using being in such early stages of development that some really weird shit happens regularly.

I suffer through the terminal more than any linux user should. Fuck with config files to no end, never satisfied with how things are working.

All because niri is worth it. Worth the added challenge and the pain

My brain is cooked. Pls fix.

**Comment by YaLTeR** [comment]
Glad you're enjoying it. :) I wonder what appeals to you about niri specifically out of other scrollable tilers? Since PaperWM is a good impl from my experience; I personally just prefer dynamic workspaces and how I handle monitors in niri. But for others it might be more important that niri lets you use layer-shell compared to PaperWM, etc.

**Reply by alicealysia** [reply]
The simple answer for me is how multi monitor reveals a collection of issues in every other scrollable tiler.
Karousel just kinda... refuses to work with it. You need to move any windows you want to work with into the floating layer.
PaperWM is a lot better, but I've still ran into some unique issues such as:
Windows at the edge of your screen drastically changing size if scaling is enabled.
My graphics tablet being mysteriously unable to interact with unfocused windows and bring them into focus.
And my personal favourite "sorry, you can't move that window to this monitor right now lol. I've just decided *no*"
So despite being very far from done... Niri is generally a more stable experience for me than paperWM 😆
At least with my specific workflow.

**Reply by YaLTeR** [reply]
I see, yeah, multi-monitor is probably the trickiest to "bolt onto" an existing compositor. Was one of my main annoyances too.

**Reply by alicealysia** [reply]
it's a big one. Speaking of the layer-shell, I just stumbled upon a neat little project called mixxc.
I'm just going to post a screenshot and leave it at that

![image](https://github.com/user-attachments/assets/bf1d51e6-3dd0-4b6e-ad96-b00ea217121f)
